src/lib.rs

```diff
@@ -16,6 +16,7 @@
 // TODO #![deny(missing_docs)]
 
 // Re-exports
+pub use algonaut_client as client;
```

Comment:
Needs to expose clients, kmd and the indexer e.g: https://github.com/i-schuetz/algorand-yew-example/blob/main/src/provider.rs#L8